welcome.md

```diff
@@ -0,0 +1,111 @@
+
+  - Editors: VS Code, Atom, Sublime.
+  - Terminals: iTerm 2, Hyper, Terminal 
+  - Harvest (native app)
+  - Slack (native app)
+  - Postman (native app)
+  - Chrome
+  
+
+## Logins
+
+### Gmail
+
+  We use Launchpad Lab gmail accounts for at work email, dev logins, and anything else work related. You should receive your login credentials on the first day.
+  
+### Github
+
+  Your GitHub account should have access to private Launchpad Lab repositories such as our opex projects. You should also have access to your client project. Work with your onboarding member if you do not have access.
+
+### Slack
+
+  We use Slack *constantly*. Slack has many channels, each with a specific purpose:
+
+  - #launchpadlab: discuss things that interest everyone. You should be in this room whenever you are working.
+
+  - #lunch_n_learn: volunteer or propose topics ideas to be presented every Wednesday for lunch. Topics can be technical, design, or anything you feel is worth sharing.
+
+  - #opex: short for Operational Excellence. Discuss ideas and updates relating to the opex effort.
+
+  - #space: share or discuss any non-client work.
+
+  - #books: what're you reading these days? Share good reads with your peeps!
+
+  - #shopping_list: we like having snacks around in the office, this channel helps narrow down what we want before ordering.
+
+  - #todays-track: what're you listening to? Share all your groovy tracks!
+
+  Lastly, you will be included in your client project's slack channel.
+
+### JIRA
+
+  Most of our client project use JIRA sprints for project management. If you don't already have a JIRA account, you can log in using your Launchpad Lab email. Your onboarding helper should assist getting you access to the sprints you need. You should have access to JIRA for your client project and potentially several internal projects.
+
+### Trello
+
+  Some of our internal or personal projects use Trello boards for project management. If you don't already have a Trello account, you can make one using your Launchpad Lab email. If there are client projects using Trello boards, your onboarding helper should assist getting you access to them.
+
+### Harvest / Forecast
+
+  We use Harvest in conjunction with Forecast to plan our project hours for each week, as well as keeping track of billable hours. You should be able to login with your Launchpad Lab email.
+
+  NOTE:
+
+  The Harvest native app is a great utility and integrates with JIRA - allowing you to start timers based on JIRA tickets. Highly recommended.
+
+# Lunch and Learn
+
+  We encourage learning. Each Wendesday we congregate for Lunch and Learn to hear a fellow collegue present a topic of interest. 
+  
+  The topic can be client related lessons, a personal project, or reflections on a conference recently attended; really, anything of interest that you feel is worth sharing.
+
+# Improve the Onboaring Experience
```

Comment:
@mrjonesbot Here is an [onboarding document](https://github.com/LaunchPadLab/process/blob/master/OnBoarding/first-day.md) we've used in the past. Feel free to link out to this and we can update. Or you can pull it into this app and I can review/update it maybe on a separate PR.